src/odemis/driver/test/orsay_test.py

```diff
@@ -1955,6 +1960,161 @@ def test_stop(self):
         with self.assertLogs(logger=None, level=logging.DEBUG):
             self.focus.stop()
 
+
+class TestFIBAperture(unittest.TestCase):
+    """
+    Tests for the Apertures on the Orsay FIB
+    """
+
+    oserver = None
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Setup the Orsay client
+        """
+        if TEST_NOHW == 1:
+            raise unittest.SkipTest(NO_SERVER_MSG)
+
+        cls.oserver = orsay.OrsayComponent(**CONFIG_ORSAY)
+        cls.datamodel = cls.oserver.datamodel
+        for child in cls.oserver.children.value:
+            if child.name == CONFIG_FIBAPERTURE["name"]:
+                cls.fib_aperture = child
+
+        cls._hybridAperture = cls.datamodel.HybridAperture
+
+    @classmethod
+    def tearDownClass(cls):
+        """
+        Terminate the Orsay client
+        """
+        cls.oserver.terminate()
+
+    def testApertureCalls(self):
+        aptr_list = self.fib_aperture._hybridAperture.AllAperturesListed.Actual
```

Comment:
What does this test tests? It seems to not test any of your code. If so, I'd suggest to drop entirely, otherwise please add a docstring to explain.